src/main/java/org/prebid/server/bidder/yieldmo/YieldmoBidder.java

```diff
@@ -78,9 +86,31 @@ private ExtImpYieldmo parseImpExt(Imp imp) throws PreBidException {
         }
     }
 
-    private Imp modifyImp(Imp imp, ExtImpYieldmo ext) {
+    private Imp modifyImp(Imp imp, BidRequest bidRequest, ExtImpYieldmo ext) {
         final YieldmoImpExt modifiedExt = YieldmoImpExt.of(ext.getPlacementId(), extractGpid(imp));
-        return imp.toBuilder().ext(mapper.mapper().valueToTree(modifiedExt)).build();
+
+        Price bidFloorPrice = Price.of(imp.getBidfloorcur(), imp.getBidfloor());
+        bidFloorPrice = BidderUtil.isValidPrice(bidFloorPrice)
+                ? convertBidFloor(bidFloorPrice, imp.getId(), bidRequest) : bidFloorPrice;
+
+        return imp.toBuilder()
+            .bidfloor(bidFloorPrice.getValue())
+            .bidfloorcur(bidFloorPrice.getCurrency())
+            .ext(mapper.mapper().valueToTree(modifiedExt))
+            .build();
+    }
+
+    private Price convertBidFloor(Price bidFloorPrice, String impId, BidRequest bidRequest) {
+        final String bidFloorCur = bidFloorPrice.getCurrency();
+        try {
+            final BigDecimal convertedPrice = currencyConversionService
+                    .convertCurrency(bidFloorPrice.getValue(), bidRequest, bidFloorCur, USD_CURRENCY);
+
+            return Price.of(USD_CURRENCY, convertedPrice);
+        } catch (PreBidException e) {
+            //If currency conversion fails, we still want to recieve the bid request.
```

Comment:
Nitpick. Typo and space
` // If currency conversion fails, we still want to receive the bid request.`

Reply:
I have fixed the above issues. Thanks for reviewing.